### PR TITLE
Documentation for IAM policy resource and header fixes to cluster resource

### PR DESCRIPTION
**Pull request by Vasundharashukla**
1. **What this PR does / why we need it**: Documentation, use-cases, resource templates for IAM policy. Minor indentation fixes for security policy resource. 

3. **Additional information**
Fixes header issue in cluster resource as well.

4. **Special notes for your reviewer**
Please review the content in iam_policy.md in docs directory.

### CODE REVIEW

**Review by shreyassreenivas — APPROVED** [review]

File: docs/resources/iam_policy.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
+---
+Title: "IAM Policy Resource"
+Description: |-
+    Creating the Tanzu Kubernetes IAM policy resource.
+---
+
+# IAM Policy
+
+The `tanzu-mission-control_iam_policy` resource enables you to add, update and delete role bindings to a particular scope for management through Tanzu Mission Control.
```
Comment (ishangupta-ds) [inline docs/resources/iam_policy.md:9]: management -> identity and access management

File: docs/resources/iam_policy.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+
+IAM policy (also known as Access Management) allows you to implement role-based access control (RBAC) in Tanzu Mission Control. For more information, see [Access Control.][access-control]
+
+[access-control]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-concepts/GUID-EB9C6D83-1132-444F-8218-F264E43F25BD.html
+
+## Policy Scope and Inheritance
+
+In the Tanzu Mission Control resource hierarchy, there are five levels at which you can specify IAM policy resources:
+- **organization** - `organization` block under `scope` sub-resource
+- **object groups** - `cluster_group` block under `scope` sub-resource
```
Comment (ishangupta-ds) [inline docs/resources/iam_policy.md:19]: Can we only keep 3 levels and specify each of the allowed blocks under a particular level in the form of a list?

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: docs/resources/iam_policy.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+
+The `tanzu-mission-control_iam_policy` resource enables you to add, update and delete role bindings to a particular scope for identity and access management through Tanzu Mission Control.
+
+IAM policy (also known as Access Management) allows you to implement role-based access control (RBAC) in Tanzu Mission Control. For more information, see [Access Control.][access-control]
+
+[access-control]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-concepts/GUID-EB9C6D83-1132-444F-8218-F264E43F25BD.html
+
+## Policy Scope and Inheritance
+
+In the Tanzu Mission Control resource hierarchy, there are five levels at which you can specify IAM policy resources:
```
Comment (ishangupta-ds) [inline docs/resources/iam_policy.md:17]: three levels

File: docs/resources/iam_policy.md [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+- **organization** - `organization` block under `scope` sub-resource
+- **object groups** - `cluster_group` and `workspace` block under `scope` sub-resource
```
Comment (ishangupta-ds) [inline docs/resources/iam_policy.md:19]: and -> or

File: docs/resources/iam_policy.md [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+- **Kubernetes objects** - `cluster` and `namespace` block under `scope` sub-resource
```
Comment (ishangupta-ds) [inline docs/resources/iam_policy.md:20]: and -> or

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

**Review by ishangupta-ds — APPROVED** [review]

**Pull request merged**